### PR TITLE
Unit tests: init Redis code

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Unit tests: init Redis code

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [x] Unit tests

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: tests/mock_classes/redis.py [hunk 1]
```diff
@@ -16,6 +16,8 @@ def test_xyz():
 
     def __init__(self, **kwargs):
         """Initialize simple dict used instead of Redis storage."""
+        self.kwargs = kwargs
+        print(kwargs)
```
Comment (vishnuchalla) [inline tests/mock_classes/redis.py:20]: Is this print just for debugging purposes?

Comment (tisnik) [reply tests/mock_classes/redis.py:20]: yes. by default we suppress prints, so it is harmles

Comment (vishnuchalla) [reply tests/mock_classes/redis.py:20]: Acknowledged. Thank you 👍🏼

Comment (bparees) [reply tests/mock_classes/redis.py:20]: see @onmete ? it's not just me who likes to print stuff in tests :)

Comment (onmete) [reply tests/mock_classes/redis.py:20]: image

**Pull request merged**